Add some more resources

matmat opened the pull request.

matmat committed:
Add some more resources

wolfgangw commented on README.md:
```diff
@@ -102,9 +110,18 @@ This list was inspired by the [awesome-broadcasting list](https://github.com/ebu
 ## News
 
 - [Celluloid Junkie](https://celluloidjunkie.com/) – news resource dedicated to the global film and cinema business.
+- [Cinema Tech Today](https://www.cinematech.today/) - Online edition of the Cinema Technology Magazine
```
Their server cert has expired 05/2023, breaks the flow somewhat?

matmat replied:
Sure, I can remove it. Still lots of useful historical articles going back a few years. Also I think IMIS who is responsible is going to have some kind of reboot. Hopefully they will renew the cert soon.

4lm replied:
For now, I suggest keeping the link but using `http` instead of `https` (there will still be a warning but not a blocking screen). If you do a quick Google search, you will notice that Google links to `http` and not `https` anymore. We could also add a "(http only)" note, something like this:

```markdown
Cinema Tech Today](http://www.cinematech.today/) (http only) - online edition of the Cinema Technology Magazine
```

Even though the page seems stale, I emailed Peter & Bob from cinematech.today to fix the issue. Maybe they'll still fix the issue before the "some kind of reboot." We could then switch back to `https`.

wolfgangw commented:
Great additions @matmat, wow.

wolfgangw commented on README.md:
```diff
@@ -84,9 +88,13 @@ This list was inspired by the [awesome-broadcasting list](https://github.com/ebu
+- [Film-Tech Digital Cinema Forum](http://www.film-tech.com/vbb/forum/main-forum) - General forum, with a focus on projection
+- [Filmvorführerforum](http://www.filmvorfuehrer.de/) - (German) General cinema tech forum
 - [DCP-o-matic User Forum](https://dcpomatic.com/forum/) – user forum of the DCP mastering software DCP-o-matic.
+- [DCP-o-matic Mailing Lists](https://lists.dcpomatic.com/postorius/lists/) - Mailing lists for DCP-o-matic.
```
Maybe the `DCP-o-matic` forum link is enough? The mailing list is essentially an archive by now.

4lm commented:
> Great additions @matmat, wow.

Yes, I totally agree - I'm very pleased with how this repo gets some traction lately!

matmat committed:
Update README.md

Change cinematech.today to use http due to expired certificate
Removed DCP-o-matic mailing list since it is not actively used anymore

matmat commented:
Thanks for the feedback. Updated the PR.

4lm commented:
LGTM :) I'll merge. I think we are good now. @wolfgangw, but if I'm wrong and you have further notice, we can adjust them later.
